src/Illuminate/LaravelSCommand.php

```diff
@@ -153,6 +153,10 @@ protected function showProtocols()
         ];
         $sockets = isset($config['server']['sockets']) ? $config['server']['sockets'] : [];
         foreach ($sockets as $key => $socket) {
+            if (isset($socket['enable']) && !$socket['enable']) {
```

Comment:
This is just to show the list of sockets, you need to add the same judgment logic [here](https://github.com/hhxsv5/laravel-s/blob/master/src/Swoole/Server.php#L127), and then explain it on the documents.

Reply:
Look forward to your commits.

Reply:
Its my mistake. I resubmitted the code